core-web/src/main/java/org/silverpeas/core/web/look/LookSilverpeasV5Helper.java

```diff
@@ -259,6 +261,19 @@ private void reloadProperties(String spaceId) {
     }
   }
 
+  private void initLayoutConfiguration() {
+    DefaultLayoutConfiguration layoutConfiguration = new DefaultLayoutConfiguration(this.resources);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "layoutConfiguration" which hides the field declared at line 90. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AHiddenFieldCheck)